hero-sync.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: Getting Started with Sync
+permalink: /hero-sync/
+---
+
+# Using Sync for your Client & Server data
+
+A set of client and server apps to demonstrate sync may be found here:
+
+* [Sample feedhenry sync-client app][sync-client-template]
+* [Sample feedhenry sync-cloud app][sync-cloud-template]
+
+
+[sync-client-template]: https://github.com/feedhenry-templates/sync-app
```

Comment:
I don't think this sync app is maintained. It isn't included in the RHMAP platform templates https://github.com/feedhenry/fh-template-apps/blob/master/global.json

Did you mean to reference https://github.com/feedhenry-templates/sync-cordova-app ?